Makefile

```diff
@@ -13,6 +13,9 @@ endif
 kops: gobindata
 	GO15VENDOREXPERIMENT=1 go install -ldflags "-X main.BuildVersion=${VERSION}" k8s.io/kops/cmd/kops/...
 
+cobra-test:
+	go test k8s.io/kops/cmd/kops
```

Comment:
There already is a test: target in the Makefile. I was thinking we can combine them later if needed, but that is probably outside the scope of this PR, and requires a broader conversation on testing.

For now - this target just runs any go tests in the cmd/kops directory.